### PR TITLE
pages*: use " " instead of "=" to separate the options from their arguments

### PR DESCRIPTION
**Pull request by vitorhcl**
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

According to the [style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#option-syntax):
> We prefer using a space instead of the equals sign (=) to separate options from their arguments (i.e. use --opt arg instead of --opt=arg) unless the program does not support it.
 
I used the regex "--[\w-]+=" to find those occurences and manually reviewed them. I didn't fix all of them yet. Also, not all of the translations were updated.

### CODE REVIEW

**Comment by vitorhcl** [comment]
> Have you checked if the commands support the options without `=`?

Well, I assumed most programs suport this, I couldn't check at the moment, but I will test when I have the opportunity.

**Review by nicokosi — DISMISSED** [review]
:+1:

**Review by kant — APPROVED** [review]
LGTM

File: pages/common/bundletool.md [hunk 1]
```diff
@@ -10,28 +10,28 @@
...
 
 - Generate APKs including only one single APK for universal usage:
 
-`bundletool build-apks --bundle={{path/to/bundle.aab}} --mode={{universal}} --ks={{path/to/key.keystore}} --ks-key-alias={{key_alias}} --output={{path/to/file.apks}}`
+`bundletool build-apks --bundle {{path/to/bundle.aab}} --mode {{universal}} --ks {{path/to/key.keystore}} --ks-key-alias {{key_alias}} --output {{path/to/file.apks}}`
 
 - Install the right combination of APKs to an emulator or device:
 
-`bundletool install-apks --apks={{path/to/file.apks}}`
+`bundletool install-apks --apks  {{path/to/file.apks}}`
```
Comment (nicokosi) [inline pages/common/bundletool.md:25]:
```suggestion
`bundletool install-apks --apks {{path/to/file.apks}}`
```

File: pages/common/bundletool.md [hunk 2]
```diff
@@ -10,28 +10,28 @@
...
 
 - Estimate the download size of an application:
 
-`bundletool get-size total --apks={{path/to/file.apks}}`
+`bundletool get-size total --apks  {{path/to/file.apks}}`
```
Comment (nicokosi) [inline pages/common/bundletool.md:29]:
```suggestion
`bundletool get-size total --apks {{path/to/file.apks}}`
```

**Review by kant — APPROVED** [review]
LGTM

**Comment by vitorhcl** [comment]
I kept the equal sign in `chroot` because I can't test it yet but this PR is ready for review, we can fix the rest in another PR

**Review by tricantivu — APPROVED** [review]

**Pull request merged**